leshan-server-core/src/main/java/org/eclipse/leshan/server/response/ResponseProcessingTask.java

```diff
@@ -89,8 +89,7 @@ private void executeAction() {
         for (ResponseListener listener : responseListeners) {
             if (listener != null) {
                 if (hasException) {
-                    LOG.debug("invoke response listener for requestTicket {} with exception {}", requestTicket,
-                            exception);
+                    LOG.debug("invoke response listener for requestTicket throws exception", requestTicket, exception);
```

Comment:
If the last param is an exception, slf4j prints since version 1.6.0 the stacktrace anyways. See http://www.slf4j.org/faq.html#paramException (exmaple is with error log, but also true for debug statements)